README.md

```diff
@@ -65,9 +48,50 @@ To build the application Docker image, and start the application container and P
 
 You should then be able to access the Prometheus dashboard on:
 
-    http://localhost:9090/graph
+    http://localhost:9090
 
 Prometheus should find and immediately start scraping metrics from the application container. You can check that it's found the application container by looking at the list of "targets" that Prometheus is scraping:
 
     http://localhost:9090/targets
 
+### Section 4: Prometheus Queries
+
+Prometheus using it's own query language called [PromQL](https://prometheus.io/docs/prometheus/latest/querying/basics/). You can enter PromQL queries in the `/graph` page of the Prometheus UI.
+
+To see the counter exported previously, we can use the PromQL query:
+
+    requests_total
```

Comment:
We don't reference this requests_total anymore before this point. Assuming we have in the workshop but to minimize confusion we need to make sure we use the same name.

Reply:
I think we do, it's here: https://github.com/ecosia/go-prometheus-workshop/pull/4/files#diff-04c6e90faac2675aa89e2176d2eec7d8R28

but yes, it could be made more clear, good catch